src/software/world/world.cpp

```diff
@@ -14,7 +14,8 @@ World::World(const Field &field, const Ball &ball, const Team &friendly_team,
       // Store a small buffer of previous referee commands so we can filter out noise
       referee_command_history_(REFEREE_COMMAND_BUFFER_SIZE),
       referee_stage_history_(REFEREE_COMMAND_BUFFER_SIZE),
-      team_with_possesion_(TeamSide::ENEMY)
+      team_with_possesion_(TeamSide::ENEMY),
+      estop_enabled(false)
```

Comment:
rather than involving sensorfusion and world, can we implement all of it in backend.cpp?

Reply:
That is possible. Only reasons against it would be that the `estop_enabled` field  would be treated differently from other fields in the in `robot status` proto, and `Backend` will become slightly more than just a way of communicating info across systems. Also not all implementations of backend will make use of `estop_enabled` . If we are fine with that, i will make the change.